packages/fxa-payments-server/src/routes/Product/PayPalButton/index.tsx

```diff
@@ -106,6 +108,7 @@ export const PaypalButton = ({
 
   const onError = useCallback(
     (error) => {
+      error.code = 'general-paypal-error';
```

Comment:
I don't know what these errors look like, but they might already have a `code` property on them?  If so, we are overwriting them.  Could you verify?